Python/Wikipedia_Scraper/main.py

```diff
@@ -0,0 +1,100 @@
+from bs4 import BeautifulSoup
+import requests
+
+
+def get_player_name():
+    """ Takes Football player's name input from the user and
+    returns a string suitable to concatenate in the wikipedia URL. """
```

Comment:
There needs to be no whitespace after the triple quotes for string, alternatively what you should prefer is a white line, what I basically mean is this - 
BAD
```
    """ Takes Football player's name input from the user and
    returns a string suitable to concatenate in the wikipedia URL. """
```

GOOD

```
    """Takes Football player's name input from the user and
    returns a string suitable to concatenate in the wikipedia URL."""
```

BEST

```
"""
Takes Football player's name input from the user and
returns a string suitable to concatenate in the wikipedia URL.
"""
```